client/coral-ui/components/Drawer.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
+import cn from 'classnames';
 import PropTypes from 'prop-types';
 import styles from './Drawer.css';
 
-const Drawer = ({children, onClose}) => {
+const Drawer = ({children, onClose, className = ''}) => {
   return (
-    <div className={styles.drawer}>
-      <div className={styles.closeButton} onClick={onClose}>×</div>
+    <div className={cn(styles.drawer, className)}>
+      {/* TODO: Swap out with button */}
```

Comment:
Let's do it :-)

Reply:
I didn't add that line. It's meant for the Button Component? 🤔

Reply:
No I think just the normal html button. I thought you added the line^^

Reply:
Done